_episodes/08-putting-it-all-together.md

```diff
@@ -129,166 +137,171 @@ writing:
 ~~~
 {: .language-python}
 
-We can start by plotting the values of a list of numbers (matplotlib can handle
-many types of numeric data, including numpy arrays and pandas DataFrames - we
-are just using a list as an example!):
+### `plt` pyplot versus object based matplotlib
+
+Consider the following example data:
 
 ~~~
-    list_numbers = [1.5, 4, 2.2, 5.7]
-    plt.plot(list_numbers)
-    plt.show()
+    import numpy as np
+    x = np.linspace(0, 5, 10)
+    y = x ** 2
 ~~~
```

Comment:
remove whitespaces in the beginnings of lines

Reply:
in `python-novice-inflammation` we agreed to not import `numpy` as `np`. I'd suggest this lesson does the same (unnecessary cognitive load)